ui/package.json

```diff
@@ -23,6 +26,7 @@
     }
   },
   "devDependencies": {
+    "@kadira/storybook": "^2.21.0",
```

Comment:
Thankfully, it looks like I remembered to check in my script to do it: `etc/licenses.sh` ... although it looks like the Go licenses are not generated correctly from that script.  Anyway, the javascript licenses use this: 
```sh
nlf -c |awk -F, '{printf "%s %s \[%s\]\(%s\)\n", $1, $2, $5, $4}'
```